README.md

```diff
@@ -117,7 +117,7 @@ jobs:
 | `test_search_query` | string  | _optional_  | Trigger tests corresponding to a [search][5] query. **Default:** none.                                                                                                                                   |
 | `subdomain`         | string  | _optional_  | The name of the custom subdomain set to access your Datadog application. If the URL used to access Datadog is `myorg.datadoghq.com`, the subdomain value needs to be set to `myorg`. **Default:** `app`. |
 | `files`             | string  | _optional_  | Glob pattern to detect Synthetic tests config files. **Default:** `{,!(node_modules)/**/}*.synthetics.json`.                                                                                             |
-| `datadog_site`      | string  | _optional_  | The Datadog site. For users in the EU, set to `datadoghq.eu`. For example: `datadoghq.com` or `datadoghq.eu`. **Default:** `datadoghq.com`.                                                              |
+| `datadog_site`      | string  | _optional_  | The [Datadog site][11]. Your Datadog site is {{< region-param key="dd_site" code="true" >}}. **Default:** `datadoghq.com`.                                                              |
```

Comment:
As per the [documentation about region shortcode](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1183023503/Regions+Sites#Site-Region-Shortcode):
> Note: This shortcode only works in the documentation and dogweb repo. For integrations-core or integrations-extras, use the site region partial.

```suggestion
| `datadog_site`      | string  | _optional_  | The [Datadog site][11]. <!-- partial Your Datadog site is {{< region-param key="dd_site" code="true" >}}. partial --> **Default:** `datadoghq.com`.                                                              |
```